EdFi.Buzz.UI/eng/edfi.buzz.ui.nuspec

```diff
@@ -13,7 +13,7 @@
   </metadata>
   <files>
     <!-- NB: nuget.exe does not handle widlcards with forward-slash paths. Must use back-slash. -->
-    <file src="..\build\**" target="build" />
+    <file src="..\**" target="src" exclude="**\node_modules\**"/>
```

Comment:
We should exclude the file that we do not need to remove this warning...
![image](https://user-images.githubusercontent.com/63315476/97366754-d8a71180-1875-11eb-8ee9-34ef939575d2.png)